Source/controls/plrctrls.cpp

```diff
@@ -107,11 +107,13 @@ int GetDistanceRanged(Point destination)
 	return plr[myplr].position.future.ExactDistance(destination);
 }
 
+/**
+ * @todo The loops in this function are iterating through tiles offset from the player position. This

```

Comment:
Instead of leaving `todo`s in the code, I'd suggest creating issues in github and documenting there instead.

TODO comments are best used for temporary work, while you are still coding the change, and should ideally not be merged.

If we add every refactoring opportunity as a TODO comment, the code will be absolutely littered with them in no time.